src/MLOps.NET.AWS/Storage/StorageAccountModelRepository.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Amazon;
+using Amazon.S3;
+using Amazon.S3.Model;
+using Amazon.S3.Util;
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace MLOps.NET.Storage
+{
+    internal sealed class StorageAccountModelRepository : IModelRepository
+    {
+        private readonly AmazonS3Client amazonS3Client;
+        private string bucketName;
+
+        public StorageAccountModelRepository(string awsAccessKeyId, string awsSecretAccessKey, string regionName, string bucketName)
+        {
+            var region = RegionEndpoint.GetBySystemName(regionName);
+            this.bucketName = bucketName;
+            this.amazonS3Client = new AmazonS3Client(awsAccessKeyId, awsSecretAccessKey, region);
```

Comment:
Could we pass in this S3Client in the ctor instead? Like that this class would be a lot easier to unit test.

Reply:
@aslotte should it be newed up in mlopsbuilderextensions instead?

Reply:
No need. We added support a couple of PR's ago to configure the MetaDataStore and ModelRepository separately, meaning that you should be able to use the AWS S3 Model Repository and e.g. Azure TableStorage for MetaData. Let me know if that clarifies it.

Reply:
i meant regarding newing up the amozonclient, if we have to pass it to ctor, then it would be newed up in extension class?

Reply:
Ah sorry, yes exactly :)